### PR TITLE
refactor: upgrade ninja and skill controllers to 1.7

### PR DESCRIPTION
**Pull request by Afonso-santos**
Upgraded ninja and skill controllers.

### CODE REVIEW

**Review by danielsp45 — DISMISSED** [review]
Run the `mix format` 🙏🏻

File: lib/bokken_web/controllers/admin/ninja/ninja_json.ex [hunk 1]
```diff
@@ -1,13 +1,13 @@
-defmodule BokkenWeb.Admin.NinjaView do
-  use BokkenWeb, :view
+defmodule BokkenWeb.Admin.NinjaJSON do
+
 
   alias Bokken.Uploaders.Avatar
 
-  def render("index.json", %{ninjas: ninjas}) do
-    %{data: render_many(ninjas, __MODULE__, "ninja.json")}
+  def show(%{ninjas: ninjas}) do
```
Comment (ruilopesm) [inline lib/bokken_web/controllers/admin/ninja/ninja_json.ex]:
```suggestion
  def index(%{ninjas: ninjas}) do
```
An index is meant to render a collection, while a show is meant to render an item.

File: lib/bokken_web/controllers/ninja/ninja_json.ex [hunk 2]
```diff
@@ -0,0 +1,60 @@
+defmodule BokkenWeb.NinjaJSON do
+
```
Comment (ruilopesm) [inline lib/bokken_web/controllers/ninja/ninja_json.ex]:
```suggestion
```

File: lib/bokken_web/controllers/ninja/ninja_json.ex [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  def index( %{ninjas: ninjas, current_user: current_user}) do
+    %{data: for(ninja <- ninjas, do: data(ninja, current_user))}
+  end
+
+  def show(%{ninja: ninja, current_user: current_user}) do
+    %{data: data(ninja, current_user)}
+  end
+
+  def ninja(%{ninja: ninja, current_user: current_user}) do
+    data(ninja, current_user)
```
Comment (ruilopesm) [inline lib/bokken_web/controllers/ninja/ninja_json.ex]:
```suggestion
    data(ninja)
```

File: lib/bokken_web/controllers/ninja/ninja_json.ex [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    |> Map.merge(personal(ninja, current_user))
+    |> Map.merge(sensitive(ninja, current_user))
+  end
+
+  defp data(ninja, _) do
```
Comment (ruilopesm) [inline lib/bokken_web/controllers/ninja/ninja_json.ex]:
```suggestion
  defp data(ninja) do
```

File: lib/bokken_web/controllers/skill/skill_json.ex [hunk 5]
```diff
@@ -0,0 +1,23 @@
+defmodule BokkenWeb.SkillJSON do
+
```
Comment (ruilopesm) [inline lib/bokken_web/controllers/skill/skill_json.ex]:
```suggestion
```

**Review by ruilopesm — APPROVED** [review]

**Pull request merged**